### ISSUE TITLE
Årsaksforhold

### ISSUE THREAD

**Issue by jeanetvi** [realfagstermer/realfagstermer#1391]
Jeg finner egentlig ingen bedre oversettelser på dette enn Causality. 
Det burde jo være kausalitet på norsk også, men kausalitet er definert som årsakssammenheng. 

---
*REAL007986 (Årsaksforhold) i [Emnesøk](https://app.uio.no/ub/emnesok/realfagstermer/search?id=c007986), [Skosmos](http://data.ub.uio.no/realfagstermer/c007986), [Okapi](http://ub-viz01.uio.no/okapi2/#/search?q=realfagstermer%3A%22%C3%85rsaksforhold%22), [Soksed](http://ub-soksed.uio.no/concepts/REAL007986)*

**Comment by Hypsibius** [comment]
Men kva er forskjellen mellom årsaksforhold og årsakssamanheng?

(spurte hun, eplekjekt)

**Comment by danmichaelo** [comment]
Ja, det var det jeg begynte å lure på også :) Mistenker at det er synonymt nok til at de kan slås sammen. Ser at Humord har gjort det samme, så prøver

```
Kausalitet
BF Årsakssammenheng
BF Årsaksforhold
```

Det er ikke en fornærmelse mot filosofifaget, @hgsl?

**Comment by Hypsibius** [comment]
Leste for ei stund sidan denne, ogh blei om mogeleg enno meir i tvil: 

![image](https://user-images.githubusercontent.com/9350370/46284642-eb589a80-c578-11e8-91d5-fdba1d0f2a9d.png)

**Comment by hgsl** [comment]
Haha, det skjønner jeg godt! Om det er til noen trøst så er en tilstand av tvil ofte et godt utgangspunkt for filosofisk tenkning - bare spør Descartes. :)
